doc/changelog.rst

```diff
@@ -21,6 +21,12 @@ Importer
 
 - Require only shop change permission to execute data imports
 
+Notify
+~~~~~~
+
+- Fix migrations. This would old projects to fake migrations until
```

Comment:
is this message correct? Looks strange, maybe `This requires old projects to fake migrations until notify.0006_shop_not_null` ?